app/Console/Commands/Init.php

```diff
@@ -32,40 +41,192 @@ public function handle(): void
+                $this->fillCredentialsAndAccess();
+            }
+
+            $this->saveReadme();
+
+            $this->info('README generated successfully!');
+
+            if ($this->emptyValuesList) {
+                $this->warn('Don`t forget to fill the following empty values:');
+
+                foreach ($this->emptyValuesList as $value) {
+                    $this->warn("- {$value}");
+                }
+            }
+        }
     }
 
     protected function createAdminUser($kebabName): void
     {
         $defaultPassword = substr(md5(uniqid()), 0, 8);
 
-        $this->publishMigration([
-            'name' => $this->ask('Please enter admin name', 'Admin'),
-            'email' => $this->ask('Please enter admin email', "admin@{$kebabName}.com"),
-            'password' => $this->ask('Please enter admin password', $defaultPassword),
+        $this->adminCredentials = [
+            'name' => $this->ask('Please enter an admin name', 'Admin'),
+            'email' => $this->ask('Please enter an admin email', "admin@{$kebabName}.com"),
+            'password' => $this->ask('Please enter an admin password', $defaultPassword),
             'role_id' => Role::ADMIN
-        ]);
+        ];
+
+        $this->publishMigration();
+    }
+
+    protected function fillReadme(): void
+    {
+        $appName = $this->argument('application-name');
+        $file = $this->loadReadmePart('README.md');
+
+        $this->setReadmeValue($file, 'project_name', $appName);
+
+        $this->readmeContent = $file;
+    }
+
+    protected function fillResourcesAndContacts(): void
+    {
+        $filePart = $this->loadReadmePart('RESOURCES_AND_CONTACTS.md');
+
+        $this->updateReadmeFile($filePart);
+    }
+
+    protected function fillResources(): void
+    {
+        $resources = [
+            'issue_tracker' => 'Issue Tracker',
+            'figma' => 'Figma',
+            'sentry' => 'Sentry',
+            'datadog' => 'DataDog',
+            'argocd' => 'ArgoCD',
+            'telescope' => 'Laravel Telescope',
+        ];
+
+        $filePart = $this->loadReadmePart('RESOURCES.md');
+
+        foreach ($resources as $key => $title) {
+            if ($this->confirm("Are you going to use {$title}?", true)) {
+                $defaultLink = ($key === 'telescope') ? $this->appUrl . '/telescope' : '';
+
+                if ($link = $this->ask("Please enter a {$title} link", $defaultLink)) {
+                    $this->setReadmeValue($filePart, "{$key}_link", $link);
+                } else {
+                    $this->emptyValuesList[] = "{$title} link";
+                }
+
+                $this->removeTag($filePart, $key);
+            } else {
+                $this->removeStringByTag($filePart, $key);
+            }
+        }
+
+        $this->setReadmeValue($filePart, 'api_link', $this->appUrl);
+        $this->updateReadmeFile($filePart);
+    }
+
+    protected function fillContacts(): void
+    {
+        $contacts = [
```

Comment:
let's move it to the class const or protected field

Reply:
Done